Add SAOCOM sensor

asolarte opened the pull request:
Hello, I leave for consideration the files to incorporate SAOCOM data processing.

asolarte committed:
Incorporación de script para ingesta de datos SAOCOM. Se modifican los siguientes archivos:

	modificado:     README.md
	modificado:     components/isceobj/InsarProc/Factories.py
	modificado:     components/isceobj/IsceProc/runPreprocessor.py
	modificado:     components/isceobj/Sensor/CMakeLists.txt
	nuevo archivo:  components/isceobj/Sensor/SAOCOM_SLC.py
	modificado:     components/isceobj/Sensor/SConscript
	modificado:     components/isceobj/Sensor/__init__.py
	modificado:     components/isceobj/StripmapProc/Factories.py
	modificado:     components/isceobj/Util/estimateoffsets/EstimateOffsets.py
	modificado:     components/stdproc/stdproc/__init__.py
	nuevo archivo:  examples/input_files/reference_SAOCOM.xml

asolarte committed:
Cambios en la lectura del polinomio para el cálculo del doppler
	modificado:     components/isceobj/Sensor/SAOCOM_SLC.py

asolarte committed:
Modificaciones en el archivo de ejemplo

	modificado:     examples/input_files/reference_SAOCOM.xml

asolarte committed:
Cambio en comentario Doppler SAOCOM_SLC.py

piyushrpt commented on examples/input_files/reference_SAOCOM.xml:
```diff
@@ -0,0 +1,46 @@
+    <property name="OUTPUT">
+        slc-acqid0000076824-a-sm8-s8dp-vv.slc
+    </property>
+    <property name="unwrapper name">snaphu</property> 
```
This unwrapper name entry here is probably a typo.

asolarte replied:
Hi, I left that tag, since it works better for me than the unwrapper that runs by default.

piyushrpt commented on components/isceobj/Sensor/SAOCOM_SLC.py:
```diff
@@ -0,0 +1,449 @@
+        """
+        Exports GeoTiff to ISCE format.
+        """
+        import gdal
```
I would change this to 
```
from osgeo import gdal
```

asolarte replied:
> @asolarte - thank you so much for this contribution. Just to understand - this is for the stripmap only correct?
> 
> Also, to let you know - if you add an unpackSAOCOM.py script to contrib/stack/stripmapStack folder - you will be able to generate stacks of coregistered SLCs as well with the stack processor.

@piyushrpt  - Yes, It's for stripmap only.
Thank you so much... I'll work on the unpackSAOCOM.py script

piyushrpt commented:
@asolarte - thank you so much for this contribution. Just to understand - this is for the stripmap only correct? 

Also, to let you know - if you add an unpackSAOCOM.py script to contrib/stack/stripmapStack folder - you will be able to generate stacks of coregistered SLCs as well with the stack processor.

asolarte committed:
Cambios sugeridos por piyushrpt. Archivos modificados:
	modificado:     components/isceobj/Sensor/SAOCOM_SLC.py
	modificado:     examples/input_files/reference_SAOCOM.xml
